keyboards/install.php

```diff
@@ -136,7 +114,7 @@ protected static function WriteWindowsBoxes() {
       return $result;
     }
 
-    protected static function WriteMacOSBoxes() {
+    protected static function WritemacOSBoxes() {
```

Comment:
Do you want to rename `legacy-keyboard-details::WriteMacOSBoxes` to be consistent?

Reply:
I won't touch the legacy code :)